### PR TITLE
use new tox-lsr plugin

### PR DESCRIPTION
**Pull request by richm**
Remove the shared scripts and instead use the new tox-lsr plugin.

### CODE REVIEW

File: .github/workflows/tox.yml [hunk 1]
```diff
@@ -2,6 +2,12 @@
 name: tox
 on:  # yamllint disable-line rule:truthy
   - pull_request
+env:
+  TOX_LSR: "git+https://github.com/richm/tox-lsr@gha-test"
```
Comment (natoscott) [inline .github/workflows/tox.yml:6]: Should this be https://github.com/linux-system-roles/tox-lsr (ah, does that need PR#1 merged there first)?

**Review by natoscott — APPROVED** [review]
LGTM.

File: .github/workflows/tox.yml (hunk 1 above)
Comment (richm) [reply .github/workflows/tox.yml:6]: > Should this be https://github.com/linux-system-roles/tox-lsr (ah, does that need PR#1 merged there first)?

Yes, that will change once I confirm that tox-lsr is working in every repo.

Comment (richm) [reply .github/workflows/tox.yml]: > Yes, that will change once I confirm that tox-lsr is working in every repo.

and now it is

**Pull request merged**